WORKSPACE

```diff
@@ -21,78 +21,143 @@ workspace(name = "vaticle_typedb_driver")
 # Load @vaticle_dependencies #
 ##############################
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "io_bazel_rules_go",
+    integrity = "sha256-fHbWI2so/2laoozzX5XeMXqUcv0fsUrHl8m/aE8Js3w=",
+    urls = [
+        "https://mirror.bazel.BUILD/github.com/bazelbuild/rules_go/releases/download/v0.44.2/rules_go-v0.44.2.zip",
+        "https://github.com/bazelbuild/rules_go/releases/download/v0.44.2/rules_go-v0.44.2.zip",
+    ],
+)
+
+http_archive(
+    name = "bazel_gazelle",
```

Comment:
Let's put it inside the same macro for `go` dependencies in `dependencies` (put both `http_archive` calls inside that macro).